lib/evss/disability_compensation_form/configuration.rb

```diff
@@ -11,7 +11,7 @@ class Configuration < EVSS::Configuration
       # @return [String] The base path for the EVSS 526 endpoints
       #
       def base_path
-        "#{Settings.evss.url}/#{Settings.evss.service_name}/rest/form526/v1"
+        "#{Settings.evss.url}/#{Settings.evss.alternate_service_name}/rest/form526/v2"
```

Comment:
Is service_name used anywhere else? Could we change service_name to be the value of alternate_service_name?

Reply:
Its still used for v1 submissions.